Add support for CCS console access for groups (#103)
openshift/gcp-project-operator: README.md

### Configmap

The controller expects to find a `ConfigMap` with the name `gcp-project-operator` inside the `gcp-project-operator` namespace.

> You can easily create one example `ConfigMap` using the following command.

Reviewer: ```suggestion
You can create one example `ConfigMap` using the following command.
```

```zsh
cat <<EOF | kubectl apply -f -
apiVersion: v1
kind: ConfigMap
metadata:
  name: gcp-project-operator
  namespace: gcp-project-operator
data:
  data: |
    billingAccount: "123456-ABCDEF-123456" # Google billing ID from https://console.cloud.google.com/billing
    parentFolderID: "123456789123"         # Google Cloud organization Parent Folder ID
    ccsConsoleAccess:

>     - example-group@xxx.com # A list of groups that you want to give CCS console access to

Reviewer: ```suggestion
    - example-group@xxx.com # A list of groups that will get access to CCS projects
```

Author: Thanks! It is much better!

>     - sd-sre-platform-gcp-access@redhat.com # A list of groups that you want to give CCS console access to

Reviewer 2: We don't want the actual group name anywhere.  Let's put an example in here.

Author: Sure, I will update it.
